mmv1/templates/terraform/examples/cloudbuildv2_connection.tf.erb

```diff
@@ -0,0 +1,13 @@
+resource "google_cloudbuildv2_connection" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+  location = "us-central1"
+  name = "my-connection"
```

Comment:
We'll need to use a `vars` entry here, so that we use a unique name for each instance of the resource: https://googlecloudplatform.github.io/magic-modules/docs/how-to/add-mmv1-test/